### PR TITLE
PFSFeedback event handler tests

### LINKED ISSUE #4507: Create test for fix in #4503
In https://github.com/raiden-network/raiden/pull/4503 a bug was fixed. However it also became clear that no tests exist. They need to be added.

### PR DESCRIPTION
**Pull request by palango**
Closed #4507 

~~⚠️ Merge after #4511~~

### CODE REVIEW

File: raiden/tests/unit/test_raiden_event_handler.py [hunk 1]
```diff
@@ -91,3 +105,181 @@ def detail_participants(_participant1, _participant2, _block_identifier, _channe
...
+    raiden = make_raiden_service_mock(
+        payment_network_address=payment_network_address,
+        token_network_address=token_network_address,
+        channel_identifier=channel_identifier,
+        partner=participant,
+    )
+
+    route = [make_address(), make_address()]
+
+    # Set PFS config but no feedback topken
```
Comment (LefterisJP) [inline raiden/tests/unit/test_raiden_event_handler.py]: typo: topken -> token

File: raiden/tests/unit/test_raiden_event_handler.py [hunk 2]
```diff
@@ -91,3 +105,181 @@ def detail_participants(_participant1, _participant2, _block_identifier, _channe
     RaidenEventHandler().on_raiden_event(
         raiden=raiden, chain_state=raiden.wal.state_manager.current_state, event=event
     )
+
+
+def test_pfs_handler_handle_routefailed_with_feedback_token():
```
Comment (LefterisJP) [inline raiden/tests/unit/test_raiden_event_handler.py]: The following four tests you are adding have a very common setup phase. Can you perhaps abstract it in a function or use one of the factories we already got?

**Comment by palango** [comment]
> Also a question that was not totally obvious by looking a bit at the code. What is the pfs feedback token? Something to do with how good the routes are? I see it's a UUID.

Here's the ADR about how the feedback works: https://github.com/raiden-network/raiden-services/blob/master/adr/002-routing-feedback.md . Let me know if that's enough or if we should extend this.

File: raiden/tests/unit/test_raiden_event_handler.py (hunk 2 above)
Comment (palango) [reply raiden/tests/unit/test_raiden_event_handler.py]: Done

**Comment by LefterisJP** [comment]
@palango for the ADR I still have some questions but will ping you in chat

File: docs/changelog.rst [hunk 3]
```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`4502` Fix a Raiden crash related to routing feedback
```
Comment (LefterisJP) [inline docs/changelog.rst]: :heart: thank you

**Pull request merged**